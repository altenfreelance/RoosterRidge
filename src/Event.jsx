import React, { Component } from "react";
import Address from "./Address";

export default class Event extends Component {
  render() {
    let { name, time, img_url, date, address } = this.props;

    return (
      <div className="col-sm-6 p-3">
        <div className="card">
          <div className="container">
            <h4>
              <b>{name}</b>
            </h4>
            <div style={{ color: "black" }}>
              <h5>{new Date(date).toDateString()}</h5>
              <h5>{time}</h5>
            </div>
            <Address address={address} />
            <img
              src={img_url}
              alt={name}
              className="border border-dark"
              style={{ width: "100%"}}
            />
          </div>
        </div>
      </div>
    );
  }



}
