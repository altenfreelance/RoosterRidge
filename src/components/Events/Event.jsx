import React, { Component } from "react";
import Address from "./Address";

export default class Event extends Component {
  render() {
    let { name, time, img_url, date, address } = this.props;

    return (
      <div className="col-sm-6 p-3">
        <div>
          <div className="container">
            <h4>
              <b>{name}</b>
            </h4>
            
            <div >
              <h5>{new Date(date).toDateString()}</h5>
              <h5>{time}</h5>
            </div>

            <Address address={address} />
            <figure>
              <figcaption>{this.props.caption}</figcaption>
              <img
                src={img_url}
                alt={name}
                className="border border-dark"
                style={{ width: "100%"}}
                caption=""
                />
            </figure>
          </div>
        </div>
      </div>
    );
  }



}
