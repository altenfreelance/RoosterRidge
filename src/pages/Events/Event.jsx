import React, { Component } from "react";
import RRImage from "../../components/RRImage";
import Address from "./Address";

export default class Event extends Component {
  render() {
    let { name, time, img_url, date, address } = this.props;

    return (
      <div className="col-sm-6 p-3">
        <div>
          <div className="container">
            <h4>
              <b>{name}</b>
            </h4>

            <div >
              <h5>{new Date(date).toDateString()}</h5>
              <h5>{time}</h5>
            </div>

            <Address address={address} />
            <figure>
              <figcaption>{this.props.caption}</figcaption>
              <RRImage
                src={img_url}
                alt={name}
                style={{ width: "100%" }}
                caption=""
              />
            </figure>
          </div>
        </div>
      </div>
    );
  }



}
