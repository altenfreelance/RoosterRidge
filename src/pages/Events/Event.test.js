import React from 'react';
import Event from './Event';

import {render, cleanup} from "@testing-library/react";
import '@testing-library/jest-dom'

const event = {
    name: "Rooster Ridge in the Park",
    date: "06/18/2020",
    time: "7:00pm",
    address: "6915 Beechmont Ave. and 6910 Salem Road.",
    caption: "Join Rooster Ridge at Beech Acres Park for great music and fun!",
    img_url: "images/cappys/cappys1.jpg"
}

afterEach(cleanup);

it('renders (matches snapshot) ', () => {
    const {asFragment} = render(
        <Event
        name={event.name}
        date={event.date}
        time={event.time}
        address={event.address}
        caption={event.caption}
        img_url={event.img_url} />
    );
    expect(asFragment()).toMatchSnapshot();
});
