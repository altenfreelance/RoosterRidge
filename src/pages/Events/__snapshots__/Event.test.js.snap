// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders (matches snapshot)  1`] = `
<DocumentFragment>
  <div
    class="col-sm-6 p-3"
  >
    <div>
      <div
        class="container"
      >
        <h4>
          <b>
            Rooster Ridge in the Park
          </b>
        </h4>
        <div>
          <h5>
            Thu Jun 18 2020
          </h5>
          <h5>
            7:00pm
          </h5>
        </div>
        <p
          class="link"
        >
          6915 Beechmont Ave. and 6910 Salem Road.
        </p>
        <figure>
          <figcaption>
            Join Rooster Ridge at Beech Acres Park for great music and fun!
          </figcaption>
          <img
            alt="Rooster Ridge in the Park"
            caption=""
            src="images/cappys/cappys1.jpg"
            style="width: 100%;"
          />
        </figure>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
